docs/deploy-ephemeral.md

```diff
@@ -0,0 +1,22 @@
+# How to deploy to ephemeral environment
+
+## Assumptions
+
+1. You've got access to Ephemeral environment.
+2. You've got [bonfire setuped-up](https://consoledot.pages.redhat.com/docs/dev/getting-started/ephemeral/onboarding.html)
+3. You've got `oc` cli tool
+4. You've logged in to Ephemeral environment through `oc login` as described in [Onboarding to Ephemeral](https://consoledot.pages.redhat.com/docs/dev/getting-started/ephemeral/onboarding.html)
+5. You've bonfire ideally in active venv
+6. You've joined https://quay.io/organization/envision org with write access
+
+# Deploy to Ephemeral
+
+1. Copy `deploy/bonfire.example.yaml` to `deploy/bonfire.yaml`
+2. set `<path_to_service_dir>` to the path to the local path of this repo
+3. run following
+
+```
+make build-podman
+podman push provisioning-backend:latest quay.io/envision/provisioning-backend
+bonfire deploy --source local --local-config-path ./deploy/bonfire.yaml provisioning --set-image-tag quay.io/envision/provisioning-backend=latest
+```
```

Comment:
I guess direct pushing to envision quay.io account is temporary solution, isn't it?

Reply:
by default, you're supposed to push a tag calculated from current commit, but I guess we should make a decision for the best solution. Might be necessary to use own quay accounts or at least personalized tags :thinking: